### PR TITLE
NuGetAuditMode default change respects SdkAnalysisLevel (#3354)

### PR DESCRIPTION
**Pull request by nkolev92**

### CODE REVIEW

**Review by JonDouglas — APPROVED** [review]

File: docs/reference/errors-and-warnings/NU1901-NU1904.md [hunk 1]
```diff
@@ -51,3 +51,6 @@ For example `<NuGetAuditSuppress Include="https://github.com/advisories/GHSA-g3q
 
 If you do not want NuGet to check for packages with known vulnerabilities during restore, add `<NuGetAudit>false</NuGetAudit>` inside a `<PropertyGroup>` in your project file, or a [`Directory.Build.props` file](/visualstudio/msbuild/customize-by-directory).
 If you would like to run NuGet Audit on developer machines, but disable it on CI pipelines, you can take advantage of MSBuild importing environment variables, and create a NuGetAudit environment variable set to `false` in your pipeline definition.
+
+In NuGet 6.12 (Visual Studio/MSBuild 17.12 and .NET 9.0.100 SDK), NuGet changed the default for `NuGetAuditMode` to `all`, which means report on transitive packages with known vulnerabilities.
```
Comment (jebriede) [inline docs/reference/errors-and-warnings/NU1901-NU1904.md:55]:
```suggestion
In NuGet 6.12 (Visual Studio/MSBuild 17.12 and .NET 9.0.100 SDK), the default setting for `NuGetAuditMode` was changed to `all`. This means that NuGet will now report on transitive packages with known vulnerabilities.
```

**Review by jebriede — APPROVED** [review]
Approved but with a suggestion on wording.

File: docs/reference/errors-and-warnings/NU1901-NU1904.md (hunk 1 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1901-NU1904.md:55]: I'll PR this suggestion to main.

Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1901-NU1904.md:55]: https://github.com/NuGet/docs.microsoft.com-nuget/pull/3357

**Pull request merged**